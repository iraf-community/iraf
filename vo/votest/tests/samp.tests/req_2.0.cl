#
#  Demonstrate general use of SAMP interoperability.

# Set the test description string.
votest.descr = "Demonstrate general use of SAMP interoperability"


print ("------------------------------------------------------------------")
print ("Req  2:  Users shall be able to interoperate with other SAMP")
print ("         (or WebSAMP) enabled applications.")
print ("------------------------------------------------------------------")


fcache init

#--------------------------------------------------------------------------

print ("Demonstration of the general use of SAMP is accomplished via the")
print ("specific SAMP test scripts.")
print ("")
print ("")
