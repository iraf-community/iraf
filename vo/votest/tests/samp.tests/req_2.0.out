------------------------------------------------------------------
Req  2:  Users shall be able to interoperate with other SAMP
         (or WebSAMP) enabled applications.
------------------------------------------------------------------
Demonstration of the general use of SAMP is accomplished via the
specific SAMP test scripts.


